api/namex/resources/name_requests/report_resource.py

```diff
@@ -241,14 +241,11 @@ def _get_next_action_text(entity_type_cd: str):
                           ' Partnership</a>'
             },
             'GP': {
-               'NEW': 'To complete your filing, visit <a href="https://www2.gov.bc.ca/gov/content/employment-business/'
-                      'business/managing-a-business/permits-licences/businesses-incorporated-companies/'
-                      'proprietorships-partnerships/registering">Registering Proprietorships and Partnerships'
-                      '</a> for more information',
-               'DEFAULT': 'To complete your filing, visit <a href="https://www2.gov.bc.ca/gov/content/'
-                          'employment-business/business/managing-a-business/permits-licences/'
-                          'businesses-incorporated-companies/proprietorships-partnerships/registering">'
-                          'Registering Proprietorships and Partnerships</a> for more information. To learn more, visit '
+               'NEW': 'To complete your filing, visit <a href="https://www.bcregistry.ca/business/auth/home/decide-business">'
+                        'BC Registeries and Online Services'
```

Comment:
`BC Registeries and Online Services` should be `BC Registries and Online Services` on L245 and 248